test/end-to-end/login.spec.ts

```diff
@@ -0,0 +1,65 @@
+import * as sinon from "sinon";
+import * as http from "http";
+import { expect } from "chai";
+
+const XMLHttpRequest = require("xmlhttprequest").XMLHttpRequest;
+
+const baseUrl = "http://localhost:8080/graphql";
+
+const payload = JSON.stringify({
+  query: `
+    query login {
+      login(email: "johnny@test.dk", password: "1234") {
+        _id
+      }
+    }
+  `,
+});
+
+const payloadWrongPassword = JSON.stringify({
+  query: `
+    query login {
+      login(email: "johnny@test.dk", password: "1111") {
+        _id
+      }
+    }
+  `,
+});
+
+describe("query Login", function () {
```

Comment:
Could use fat-arrow ( `() => {}` ) instead of `function`
Same with the rest of them.